Analysis/Utkscan/processors/include/PspmtProcessor.hpp

```diff
@@ -76,7 +76,7 @@ class PspmtProcessor : public EventProcessor {
     /// will use to calculate the position.
     ///@return The x,y position of the interaction
     std::pair<double, double> CalculatePosition(double &xa, double &xb, double &ya,
```

Comment:
Can mark const and possibly mark noexcept if we can guarantee that none of the combinations of the denominators will ever be 0

Reply:
Fixed const correctness, not marking noexcept because the check for zero denom is not easy and is better to just eat the NaN/Inf we would get out of divide by zero